ckanext/dcatapchharvest/tests/test_dcatap_ch_parse_deprecated_rdf.py

```diff
@@ -1,29 +1,22 @@
-# -*- coding: utf-8 -*-
-
 import json
 
-import nose
-
 from ckanext.dcat.processors import RDFParser
 from ckanext.dcatapchharvest.tests.base_test_classes import BaseParseTest
 
-eq_ = nose.tools.eq_
-assert_true = nose.tools.assert_true
-
 
-class DeprecatedProfileParseTest(BaseParseTest):
+class TestDeprecatedProfileParse(BaseParseTest):
     def test_deprecated_landing_page_import(self):
         contents = self._get_file_contents("deprecated/dataset-landing-page.xml")
         p = RDFParser(profiles=["swiss_dcat_ap"])
         p.parse(contents)
         dataset = [d for d in p.datasets()][0]
-        eq_(dataset["url"], "https://www.bfs.admin.ch/bfs/de/home/statistiken.html")
+        assert dataset["url"] == "http://www.bafu.admin.ch/laerm/index.html?lang=de"
 
     def test_deprecated_publisher_import(self):
         contents = self._get_file_contents("deprecated/dataset-publisher.xml")
         p = RDFParser(profiles=["swiss_dcat_ap"])
         p.parse(contents)
         dataset = [d for d in p.datasets()][0]
         publisher = json.loads(dataset["publisher"])
-        eq_(publisher["name"], "Landesamt Topographie Swisstopo")
-        eq_(publisher["url"], "https://swisstopo")
+        assert publisher["name"] == "Bundesamt für Landestopografie swisstopo"
+        assert publisher["url"] == ""
```

Comment:
As in comment above: the values in the fixture file https://github.com/opendata-swiss/ckanext-dcatapchharvest/blob/master/ckanext/dcatapchharvest/tests/fixtures/deprecated/dataset-publisher.xml are not the same as used to be asserted in this test. :-/ I have to check whether the tests in this file were ever actually run...

Reply:
Oh, right, I know that these tests weren't run, because I had to change their class names to get them to be recognised by `pytest`. I guess someone just added them and then assumed that because nothing broke, they were passing.

I will add `-v` into the `pytest` command in `test.yml` so it outputs which tests it's actually run, because we've had this problem occur before.